lib/bolt/executor.rb

```diff
@@ -485,21 +485,29 @@ def wait_until(timeout, retry_interval)
 
     def prompt(prompt, options)
       @prompting = true
+
       unless $stdin.tty?
```

Comment:
Hm, I don't entirely grok this behavior but setting `@prompting` to true when `$stdin.tty?` is false seems to cause this to spin forever. If we move this check to be before `@prompting = true` then this succeeds as expected.